app/lib/pdf_filler/md502_pdf.rb

```diff
@@ -85,5 +90,21 @@ def filing_status(method)
     def checkbox_value(value)
       value.present? ? 'Yes' : 'Off'
     end
+
+    def generate_codes_for_502_su
+      calculated_fields_code_letters = {MD502_SU_LINE_AB: "ab", MD502_SU_LINE_U: "u", MD502_SU_LINE_V: "v"}
```

Comment:
[specs of dust] I think this would be slightly better with newlines for each pair (esp because the list may grow in the future), but incredibly minor.
```suggestion
      calculated_fields_code_letters = {
        MD502_SU_LINE_AB: "ab",
        MD502_SU_LINE_U: "u",
        MD502_SU_LINE_V: "v"
      }
```

Reply:
happy to do that :)